test/test_ufl_wrapper.py

```diff
@@ -42,9 +42,9 @@ def test_create_tensor_element(inputs):
     (basix.ElementFamily.P, "triangle", 2),
 ])
 def test_tensor_element_hash(inputs):
-    e = ufl_wrapper.create_element(*inputs)
-    sym = ufl_wrapper.TensorElement(e, symmetric=True)
-    asym = ufl_wrapper.TensorElement(e, symmetric=None)
+    e = basix.ufl.element(*inputs)
+    sym = basix.ufl.BlockedElement(e, shape=(2, 2), symmetry=True)
+    asym = basix.ufl.BlockedElement(e, shape=(2, 2), symmetry=False)
```

Comment:
It seems a bit inconsistent to use `CamelCase` for `BlockedElement` but not for the creation of a `BasixElement`.

I am wondering, do we have a similar wrapper from `BlockedElement` called `blockedelement`? 
Having different kinds of casing for initialization of elements seems a bit confusing.

Reply:
I don't like this either. It's because some of these are directly initing the class, others are using convenience functions.

I think it makes sense to add a couple more convenience functions and encourage people to not use the classes directly